package.json

```diff
@@ -43,6 +43,7 @@
     "prepublishOnly": "rm -rf src && cp -R app/javascript/active_admin src && cp -R app/assets/stylesheets/active_admin src/scss"
   },
   "dependencies": {
+    "@rails/ujs": "^7.1.3-4",
```

Comment:
this can be `7.1.400`

Reply:
I've made this change, thank you. I think they messed up on versioning since installing should have used this one but it didn't. It doesn't seem this version is tagged as the latest in NPM, FYI.